### DISCUSSION TITLE
Unable to load weights from pytorch checkpoint file

### DISCUSSION THREAD

**Discussion by En-J-A** [aub-mind/arabert#128] [General]
When I run a code to train a model in google colab, I got this kind of error

```
File "/usr/local/lib/python3.7/dist-packages/transformers/modeling_utils.py", line 1286, in from_pretrained
    f"Unable to load weights from pytorch checkpoint file for '{pretrained_model_name_or_path}' "
OSError: Unable to load weights from pytorch checkpoint file for 'model_name' at '/root/.cache/huggingface/transformers/8e7b2a80ddcb5611b27d8c89e1e8e33a947e105415051402a22b9c8d7d1caeb0.e22331f3a065b885b30ae3dd1ff11ccaf7fbc444485f6eb07ef5e0138bca8b70'If you tried to load a PyTorch model from a TF 2.0 checkpoint, please set from_tf=True. 
[ ]
```
where 'model_name' is a model exist in huggingface

But , when I run the same code on my local machine this error doesn't occur and the weights have loaded.

If you know the reason or how can I solve this issue, please guide me?

**Comment by WissamAntoun** [comment]
model_name should be replace by the model path in the hugging face library for example model_name='aubmindlab/bert-base-arabertv02'

https://huggingface.co/models?filter=ar

**Reply by En-J-A** [reply]
> mple model_name='aubmindlab/bert-base-arabertv02'

I already use like this, but it does not work in Colab.

**Reply by WissamAntoun** [reply]
can you copy paste the related code here? also which version of transformers you are using

**Reply by En-J-A** [reply]
transformers==2.8.0
tokenizers==0.5.2
torch==1.4.0

**Reply by En-J-A** [reply]
I am not using arabert , I used both Bart  and t5 model , Does this error relate to the model ?
```

tokenizer = MBart50TokenizerFast.from_pretrained(args.model_path if test else 'facebook/mbart-large-50',force_download=True)
model =  MBartForConditionalGeneration.from_pretrained(args.model_path if test else 'facebook/mbart-large-50')
```

**Reply by WissamAntoun** [reply]
try using the latest version of transformers

**Reply by En-J-A** [reply]
> try using the latest version of transformers

Ok. I updated all the libraries to the last versions and it works.
thank you very much
